data/events.json

```diff
@@ -0,0 +1,558 @@
+  },
+  {
+    "eventDbId": "6000000049",
+    "studyDbId": "6000000010",
+    "date": {
+      "discreteDates": ["2017-04-20"]
+    },
+    "eventDescription": "Field planted with Kincaid planter, 5 cm seed spacing (planned for thinning to 14cm spacing after seedling emergence).  Planted plot length 3.5 m with 0.5 m alley length, 2-row plots spaced at 76.2 cm.  Seed depth 3.18 cm. ",
+    "eventParameters": [
+        {
+          "code": "PLMA",
+          "name": "Planting Method",
+          "value": "S",
+          "unit":"code",
+          "codeValueDescription": "Dry Seed"
+        },
+        {
+          "code": "CRID",
+          "name": "Crop (or weed) species identifier",
+          "value": "SGG",
+          "unit": "code",
+          "codeValueDescription": "Sorghum bicolor"
+        },
+        {
+          "code": "PLSL",
```

Comment:
I'm not seeing this code in the ICASA data dictionary?

Reply:
Or PLAL, PLNR, and PLSD, which are listed just below this entry.

Reply:
I think I have fixed these. Hopefully it isn't something that the AI code assistants (GitHub Copilot) injected!!!